examples/step-97/doc/intro.dox

```diff
@@ -0,0 +1,938 @@
+$\vec{J}_f$, equals zero at the boundary $\Gamma_{R1}$ by definition of the
+problem. We will neglect this integral in the computer code.
+
+<h4>Converting potentials into fields</h4>
+
+After the magnetic vector potential is computed, it needs to be converted into
+the magnetic field as
+\f[
+\vec{B} = \vec{\nabla} \times \vec{A}.
+\f]
+The problem of computing this equation can be replaced by the problem of
+minimizing the following functional:
+\f[
+F(\vec{B}) = \iiint_{\Omega} \big| \vec{B} \big|^2 dV -
+2 \iiint_{\Omega} \bigg( \vec{\nabla} \times \vec{A} \bigg) \cdot \vec{B} dV.
+\f]
+This functional is minimized by solving a system of linear equations with the
+system matrix
+\f[
+A_{ij} = \underbrace{\iiint_{\Omega} \vec{N}_i \cdot \vec{N}_j dV}_{I_a}
+\f]
+and right-hand side column vector
+\f[
+b_i = \underbrace{\iiint_{\Omega} \bigg( \vec{\nabla} \times \vec{A} \bigg)
+\cdot \vec{N}_i dS}_{I_b}.
+\f]
+This time, however, $\vec{N}_i$ are the shape functions of the FE_RaviartThomas
+finite elements. The vector field $\vec{A}$ in the last equation is the numerically
+computed magnetic vector potential, i.e., a linear combination of the shape
+functions of the FE_Nedelec finite elements.
+
+It is informative to verify the quality of the computed current vector potential,
+$\vec{T}$. We can do this by converting it back into the free-current
+density, $\vec{J}_f$, and comparing the result to the closed-form analytical
+expression given by the definition of the problem, see above. The current vector
+potential relates to the free-current density as
+\f[
+\vec{J}_f = \vec{\nabla} \times \vec{T}.
+\f]
+This equation is, essentially, the same as the equation for the converting
+$\vec{A}$ into $\vec{B}$, see the first equation of this section. Consequently,
+we can simply adapt the equations for the functional, system matrix, and the
+right-hand side given above by making the following substitutions:
+\f{equation}{
+\begin{array}{lcr}
+\vec{A} \rightarrow \vec{T} &\text{ and }& \vec{B} \rightarrow \vec{J}_f.
+\end{array}
+\f}
+
+The two conversions, $\vec{A}$ into $\vec{B}$ and $\vec{T}$ into $\vec{J}_f$,
+can be done by the same piece of code. We will call it a projector from
+$H(\text{curl})$ to $H(\text{div})$.
+
+<h3>Selecting finite elements</h3>
+
+The preceding sections leave an impression that we are going to model $\vec{A}$
+and $\vec{T}$ by the FE_Nedelec finite elements, while $\vec{B}$ and $\vec{J}_f$
+by the FE_RaviartThomas finite elements. It makes sense to discuss how this
+kind of choices can be made.
+
+The most simple method of assigning a particular type of finite elements to a
+physical quantity studied in electromagnetics is a contemplation of the Bossavit's
+diagram. It is presented below. The Bossavit's diagram describes the relations
+between physical quantities as they are given by the Maxwell's equations and
+constitutive relations. Most equations derived from the Maxwell's equations
```

Comment:
It's either "the Maxwell equations" or "Maxwell's equations". Either is fine, but not "the Maxwell's equations".
```suggestion
between physical quantities as they are given by Maxwell's equations and
constitutive relations. Most equations derived from Maxwell's equations
```